traffic_ops/traffic_ops_golang/deliveryservice/autorenewcerts.go

```diff
@@ -190,8 +190,12 @@ func RunAutorenewal(existingCerts []ExistingCerts, cfg *config.Config, ctx conte
 
 	if cfg.SMTP.Enabled && cfg.ConfigLetsEncrypt.SendExpEmail {
 		errCode, userErr, sysErr := AlertExpiringCerts(keysFound, *cfg)
-		if userErr != nil || sysErr != nil {
-			log.Errorf("cert autorenewal: sending email: errCode: %d userErr: %s sysErr: %s", errCode, userErr.Error(), sysErr.Error())
```

Comment:
you could instead just change the format options for errors to `%v` and get rid of the method call to `error.Error`:
```go
log.Errorf("cert autorenewal: sending email: errCode: %d userErr: %v sysErr: %v", errCode, userErr, sysErr)

Reply:
well thats a lot easier. done